src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6171,8 +6171,22 @@ namespace HVACVariableRefrigerantFlow {
 			AverageUnitMassFlow = ( PartLoadRatio * CompOnMassFlow ) + ( ( 1 - PartLoadRatio ) * CompOffMassFlow );
 			AverageOAMassFlow = ( PartLoadRatio * OACompOnMassFlow ) + ( ( 1 - PartLoadRatio ) * OACompOffMassFlow );
 		} else {
-			AverageUnitMassFlow = CompOnMassFlow;
-			AverageOAMassFlow = OACompOnMassFlow;
+			if ( VRF( VRFTU( VRFTUNum ).VRFSysNum ).VRFAlgorithmTypeNum == AlgorithmTypeSysCurve ) {
```

Comment:
@Nigusse 

I just pushed a commit, in which I put 

                        if( PartLoadRatio == 0 ) {
                               CompOffMassFlow = OACompOffMassFlow;

So there should be no conflict to delete that if-else statement.